csharp-read-write-sheet/Program.cs

```diff
@@ -19,12 +19,10 @@ class Program
 
         static void Main(string[] args)
         {
-            // Get API access token from App.config file or environment
-            string accessToken = ConfigurationManager.AppSettings["AccessToken"];
+            // Get API access token from environment variable SMARTSHEET_ACCESS_TOKEN
+            string accessToken = Environment.GetEnvironmentVariable("SMARTSHEET_ACCESS_TOKEN");
             if (string.IsNullOrEmpty(accessToken))
-                accessToken = Environment.GetEnvironmentVariable("SMARTSHEET_ACCESS_TOKEN");
-            if (string.IsNullOrEmpty(accessToken))
-                throw new Exception("Must set API access token in App.conf file");
+                throw new Exception("Must set API access token in SMARTSHEET_ACCESS_TOKEN environment variable");
 
             // Initialize client
             SmartsheetClient smartsheet = new SmartsheetBuilder().SetAccessToken(accessToken).Build();
```

Comment:
Actually, the builder itself will load the SMARTSHEET_ACCESS_TOKEN environment variable. This code can just be changed to 
```SmartsheetClient smartsheet = new SmartsheetBuilder().Build();``` 
But I still think we should specify in the comment where it is coming from.